cloudformation_templates/alarms/HTTP5xxCountAlarm.j2

```diff
@@ -0,0 +1,5 @@
+{% extends "alarms/base_alarm.j2" %}
+
+{% block description %}Login failure rate too high{% endblock %}
+
+{% block manual_url %}https://github.gds/pages/gds/digitalmarketplace-manual/alerts.html#login-failure-rate{% endblock %}
```

Comment:
Are we going to get these things written up in the manual at some point? Seems pointless to keep the URLs in otherwise.